### PR TITLE
Fd 1089 extend setupsim

### PR DESCRIPTION
**Pull request by stackdump**
Tweak SimSetup to allow Fnode0 to be demoted and replaced as a Follower
This also swaps the identity - net gain is to allow a test where fnode0 isn't special

### CODE REVIEW

**Comment by stackdump** [comment]
I also renamed one of the options in ./test.sh  changed from 'long' to 'full' 
+ added a usage helper

File: wsapi/debugapi.go [hunk 1]
```diff
@@ -427,15 +427,16 @@ func runCmd(cmd string) {
 
 	return
 }
+
 func HandleSimControl(state interfaces.IState, params interface{}) (interface{}, *primitives.JSONError) {
-	droprate := new(GetCommands)
-	err := MapToObject(params, droprate)
+	cmdLines := new(GetCommands)
```
Comment (stackdump) [inline wsapi/debugapi.go]: renamed b/c it didn't look like we are limited to droprate commands here

File: testHelper/simulation.go [hunk 2]
```diff
@@ -129,105 +130,175 @@ func StartSim(GivenNodes string, UserAddedOptions map[string]string) *state.Stat
...
-	if len(creatingNodes) > nodes {
-		t.Fatalf("Should have allocated %d nodes", len(creatingNodes))
-	}
-	WaitForMinute(state0, 1)
+func promoteNodes(creatingNodes string) int {
+
 	for i, c := range []byte(creatingNodes) {
 		fmt.Println("it:", i, c)
 		switch c {
-		case 'L', 'l':
```
Comment (stackdump) [inline testHelper/simulation.go]: I don't think we use lower anywhere - moving to try to standardize on caps

File: wsapi/debugapi.go (hunk 1 above)
Comment (factom-clay) [reply wsapi/debugapi.go]: good change

File: testHelper/simulation.go (hunk 2 above)
Comment (factom-clay) [reply testHelper/simulation.go]: What's the benefit to being case sensitive?

**Pull request merged**